### PR TITLE
Config (ESLint, Pre-commit, README)

### LINKED ISSUE #19: [FR] Install ESLint for Consistence Coding Style
# ESLinting

### Description of the feature
* Need code consistency

### Tasks to complete
- [ ] Install ESLint, config to VSCode
- [ ] Decide rules

### LINKED ISSUE #21: [FR] README
Readme: https://www.notion.so/README-6d735f16f87a469ba08fbf0b91ec2324

### PR DESCRIPTION
**Pull request by cadrianxyz**
# Description

- ES Lint for Vue
- Readme

Fixes #19  and #21 

## Type of Change

Please delete options that are not relevant.

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] Code follows the project's style guidelines **(code generates the project's style guidelines 😃)**
- [x] I have performed a self-review of the Code being pushed
- [x] Code has been reviewed by at least one other developer
- [x] Code has appropriate commenting, particularly in hard-to-understand areas
- [x] Corresponding changes to the documentation have been made
- [x] The changes generate no new warnings

### CODE REVIEW

**Comment by cadrianxyz** [comment]
@kelvin-muk please check the readme. You can view it easily [here](https://github.com/onlinkers/event-discovery-prototype/blob/88617a2c94ae27021f4ca1c10658e35d0b0e9d78/README.md) or on [notion](https://www.notion.so/README-6d735f16f87a469ba08fbf0b91ec2324)

**Comment by cadrianxyz** [comment]
@kraytaps Unfortunately ESLint doesn't support checking the <style> tags in `.vue` files. There's this [other dependency](https://stylelint.io/) that does, but idk if we want to switch to it if we are all using esLint by default, esp on VSCode

File: README.md [hunk 1]
```diff
@@ -1,29 +1,93 @@
...
+
+<hr />
+
+### Environmental Files
+
+A dotenv (`.env.local`) environmental file needs to be created at `root`
+
+> You can do this easily by copying and renaming `.env.default`
+
+### Components, Services, and Styles
```
Comment (cadrianxyz) [inline README.md]:
```suggestion
### Components and Services
```

File: README.md [hunk 2]
```diff
@@ -1,29 +1,93 @@
...
+
+None yet. Application's `master` branch is currently built on Netlify for live viewing
+
+Run `npm run build` to minify and compile the application for production
+
+## Contributing
+Contributions are only accepted from members of the [team](https://github.com/orgs/onlinkers/people). Instructions to start contributing are as follows:
+
+1. Clone the remote repository into a local environment
+2. Setup the repository (instructions [here](## Getting Started))
```
Comment (cadrianxyz) [inline README.md]:
```suggestion
2. Setup the repository (instructions [here](##getting-started))
```
Same-page links are specific format lol

**Comment by ad2969** [comment]
> @kraytaps Unfortunately ESLint doesn't support checking the <style> tags in `.vue` files. There's this [other dependency](https://stylelint.io/) that does, but idk if we want to switch to it if we are all using esLint by default, esp on VSCode

**Review by kraytaps — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,29 +1,93 @@
...
+
+Styles are written **inside** `.vue` components, recommended as scoped styles, and in SCSS
+
+Styles are pre-compiled using [sass-loader](https://vue-loader.vuejs.org/guide/pre-processors.html#sass)
+
+Overwrites to libraries or global style variables and constants can be wrriten in `src/assets/scss`, which is pre-pended using `sass-loader`
+
+## Deployment
+
+None yet. Application's `master` branch is currently built on Netlify for live viewing
```
Comment (cadrianxyz) [inline README.md]: Netlify build settings can be accessed only through the onlinker-admin organization account

**Pull request merged**